### ISSUE TITLE
Support puppeteer v5

### ISSUE THREAD

**Issue by stephenmathieson** [dequelabs/axe-core-npm#105]
Puppeteer v5 has been out for awhile now. The `@axe-core/puppeteer` package has a [`peerDependency` on puppeteer v1/v2](https://github.com/dequelabs/axe-core-npm/blob/v4.0.0/packages/puppeteer/package.json#L47-L49).

Is there a plan to support v5? Will this happen any time soon?

Installing the `@latest` tag of both yields this warning:

```
warning " > @axe-core/puppeteer@4.0.0" has incorrect peer dependency "puppeteer@^1.10.0 || ^2.0.0".
```

**Comment by stephenmathieson** [comment]
```
warning " > @axe-core/puppeteer@4.0.0" has incorrect peer dependency "puppeteer@^1.10.0 || ^2.0.0".
```

**Comment by michael-siek** [comment]
I can take a look at it.

**Comment by stephenmathieson** [comment]
Thanks Michael!

**Comment by padmavemulapati** [comment]
Verified with the latest code `axe-core-npm`, When tried to install puppeteer , it is installing with the version `puppeteer@5.4.0 ` and with this version , able to execute unit tests successfully, as well tested an example test file with simple analyse API. Got the expected results:
![image](https://user-images.githubusercontent.com/47409579/97313055-36465880-188c-11eb-8175-23e9a17d0697.png)
[output_puppeteerV5.txt](https://github.com/dequelabs/axe-core-npm/files/5445862/output_puppeteerV5.txt)
